content/6_5_building-envelope.html

```diff
@@ -92,8 +92,8 @@
                     </tr>
                     <tr>
                         <td>Simulation inputs consistent with CF</td>
-                        <td><a href="#be08p-"><a href="#be12">BE04-P</a></a></td>
-                        <td><a href="#be08b-">BE08-B,<a href="#be12">BE12-B</a></a></td>
+                        <td><a href="#be08"><a href="#be12">BE04-P</a></a></td>
```

Comment:
anchor tag nested within another anchor tag

one of them links to #be12 with BE04-P as the text